Cargo.toml

```diff
@@ -26,5 +26,6 @@ hifitime = "3.8.4"
 thiserror = "1"
 serde = { version = "1.0", optional = true, default-features = false, features = ["derive"] }
 itertools = "0.11.0"
+nyx-space = { version = "2.0.0-alpha.2", optional = true }
```

Comment:
@ChristopherRabotin,

which version do you recommend using, knowing that this crate is work in progress and having things under development is not a problem

Reply:
You choose the right one : alpha 2 is the more stable so far. The beta dev version should be stable but I don't test it enough to be confident. I am happy to try to maintain the updates, especially the one to 2.0-beta, because that will introduce a lot of changes